src/Utilities/Compiler/Options/AggregateCategorizedAnalyzerConfigOptions.cs

```diff
@@ -15,14 +15,14 @@ namespace Analyzer.Utilities
     using static CategorizedAnalyzerConfigOptionsExtensions;
 
     /// <summary>
-    /// Aggregate analyzer configuration options:
-    ///
+    /// <para>Aggregate analyzer configuration options:</para>
+    /// <para>
     /// <list type="number">
```

Comment:
⚠️ `<list>` should not be placed within `<para>`. If this change was automated, it is likely a bug in the tool.